### ISSUE TITLE
interfaces may not include member variables not flagged as error

### ISSUE THREAD

**Issue by SignpostMarv** [vimeo/psalm#1666]
* https://psalm.dev/r/9ac546b8ed
* using 7601921
* https://3v4l.org/ThdMS

```php
<?php
interface foo {
  public static $foo = ['bar'];
}
```
> Fatal error: Interfaces may not include member variables in /in/ThdMS on line 3

**Comment by muglug** [comment]
Thanks - as a general rule, things that can be caught by `php -l` aren't things that I care to detect (though Psalm will detect some of them anyway thanks to PHP Parser).
